js/templates/search.es6.js

```diff
@@ -6,7 +6,7 @@ module.exports = function () {
           <input type="text" autocomplete="off" placeholder="Search DuckDuckGo" name="q" class="search-form__input js-search-input" value="${this.model.searchText}" />
           <input class="search-form__go js-search-go" tabindex="2" value="" type="button" />
           <input type="submit" class="search-form__submit" />
-          <span class="ddg-logo"></span>
+          <span class="ddg-logo-${this.model.browser}"></span>
```

Comment:
... now you can do `<span class="ddg-logo ddg-logo--${this.model.browser}"... ` here.